prac_06/guitars.py

```diff
@@ -0,0 +1,37 @@
+"""
+List inputted guitars, including guitar position in list, name, year made, and cost, using string formatting.
+
+Guitars. Created by Malia, April 2021.
+"""
+
+from prac_06.guitar import Guitar
+
+
+def main():
+    """Print list of guitars, including guitar position in list, name, year made, and cost."""
+    guitars = []
+    print("My guitars!")
+    name = input("Name: ")
+    while name != "":
+        year = int(input("Year: "))
+        cost = float(input("Cost: $"))
+        add_guitar = Guitar(name, year, cost)
+        guitars.append(add_guitar)
```

Comment:
You could combine these to not need a variable.
`guitars.append(Guitar(name, year, cost))`